Example-Audit-REXX/RXAUDIT1.rexx

```diff
@@ -0,0 +1,2478 @@
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'number-reserved-ifl-processors'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'number-ifl-cores'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'number-reserved-ifl-cores'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'initial-processing-weight'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'initial-processing-weight-capped'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'initial-vfm-storage'
+LPARAttribute.attrI.0 = 0
+attrI = attrI + 1
+LPARAttribute.attrI = 'maximum-vfm-storage'
+LPARAttribute.attrI.0 = 0
+
+attrI = attrI + 1
+LPARAttribute.attrI = storageCentralAllocProp
+LPARAttribute.attrI.0 = 1 /* >0 indicates it's a complex property */
+
+attrI = attrI + 1
+LPARAttribute.attrI = storageExpandedAllocProp
+LPARAttribute.attrI.0 = 1 /* >0 indicates it's a complex property */
+
+/* total number of LPAR attributes to query
+   NOTE: this is not the total number of properties in the csv file
+         because each complex property may result in one or more
+         properties
+*/
+LPARAttribute.0 = attrI
+
+return  /* end function */
+
+/*************************************************/
+/* Function:  foundJSONvalue()                   */
+/* Return true if the content is not empty       */
+/*************************************************/
+foundJSONValue:
+parse arg jsonValueArg
+
+if jsonValueArg = '' | jsonValueArg = '(not found)' then
+  return FALSE
+
+return TRUE
+
+/*************************************************/
+/* Function:  EntryArrow                         */
+/* Print entry arrow '--------------->'          */
+/*************************************************/
+EntryArrow:
+say
+say '--------------->'
+return
+
+/*************************************************/
+/* Function:  ExitArrow                          */
+/* Print entry arrow '<---------------'          */
+/*************************************************/
+ExitArrow:
+say '<---------------'
+say
+return
+
+/*************************************************/
+/* Function:  WriteToFile()                      */
+/* This will write out the REXXWRT stem contents */
+/*************************************************/
+WriteToFile:
+parse arg file
+
+rc = 0
+if ISVREXX then
+  do
+    "ALLOC F(MYOUTD) DSN("||file||") SHR"
+    Address MVS "EXECIO * DISKW MYOUTD (STEM REXXWRT. FINIS"
+    "FREE F(MYOUTD)"
+  end
+else
+  do
+    Address TSO
+    "ALLOC F(MYOUTD) DSN("||file||") SHR REU"
+    "EXECIO * DISKW MYOUTD (STEM REXXWRT. FINIS"
+    "FREE F(MYOUTD)"
+  end
+
+if rc <> 0 then
+  do
+    errMsg = '** fatal error, rc=('||rc,
+             ||') encountered trying to write out content **'
+    return fatalError(errMsg)
+  end
+
+return  rc /* end function */
+
+/*************************************************/
+/* Function:  VerifyDataSet()                    */
```

Comment:
The comment should be updated to reflect the userDataSet being passed to the function

Reply:
good call, done